querydsl-core/src/main/java/com/querydsl/core/support/CollectionAnyVisitor.java

```diff
@@ -110,7 +110,7 @@ protected Predicate exists(Context c, Predicate condition) {
             Path<?> parent = (Path<?>) expr.getMetadata().getParent().accept(this, context);
             expr = new PathImpl<Object>(expr.getType(), PathMetadataFactory.forCollectionAny(parent));
             EntityPath<?> replacement = new EntityPathBase<Object>(expr.getType(),
-                    ExpressionUtils.createRootVariable(expr));
+                    ExpressionUtils.createRootVariable(expr, replacedCounter++));
```

Comment:
I do prefer to minimize the number of statements in a single line but I think that this particular instance is probably fine.

#1192 might be a good place to discuss coding style.